test/unit/utility/char_operations/char_predicate_test.cpp

```diff
@@ -10,13 +10,17 @@
 #include <string>
 
 #include <seqan3/alphabet/aminoacid/aa27.hpp>
+#include <seqan3/alphabet/concept.hpp>
```

Comment:
Is this also needed if we remove `is_in_alphabet`?

Reply:
Yes, in the very last test we use `char_is_valid_for` which is defined in `seqan3/alphabet/concept.hpp`.

Reply:
Thinking about it again, I could also remove the last test and the concept include, as we have plenty of `char_is_valid_for` tests already. It was introduced because we had an issue with the gapped alphabets not being recognised correctly by `is_in_alphabet`.

Reply:
No it's fine like this I think. One test more does not hurt :)